README.md

```diff
@@ -70,6 +72,8 @@ format: String, image format, 'png' or 'jpg', Default is 'png'
 
 processor: String, css processor, 'css' or 'less' or 'sass' or 'scss', Default is 'css'
 
+retina: Boolean, returns higher quality image, Default is false
```

Comment:
What do you think about passing in a `dotsPerPixel` (or `dppx`) of 2 instead? It matches CSS media query naming conventions (https://www.w3.org/TR/css3-values/#resolution)